src/encoded/commands/deploy.py

```diff
@@ -1039,6 +1041,12 @@ def hostname(value):
         type=int,
         help="Remote indexing"
     )
+    parser.add_argument(
```

Comment:
This argument will only work for non demos.  If I set it to three for a cluster build then chunk size is three.  But for a demo it will be 256.

Which is fine, it's just a confusing way to make defaults.  And wouldn't be the only variable in here that doesn't work the way someone would expect.  But will be cleared up if we move to using cfg directly per the other comment.

Reply:
yeah that is very sloppy.